selene/src/roblox/base.toml

```diff
@@ -467,6 +478,14 @@ required = false
 type = "number"
 required = false
 
+[[Rect.new.args]]
+type = { display = "Vector3" }
+required = false
+
+[[Rect.new.args]]
+type = { display = "Vector3" }
+required = false

```

Comment:
This also doesn't make sense.

These aren't alternate definitions, they're all additional arguments to the same function.